osu.Game.Tests/Gameplay/TestSceneStoryboardSamples.cs

```diff
@@ -70,6 +73,37 @@ public void TestSamplePlaybackAtZero()
             AddUntilStep("sample playback succeeded", () => sample.LifetimeEnd < double.MaxValue);
         }
 
+        [Test]
+        public void TestSamplePlaybackWithRateMods()
+        {
+            GameplayClockContainer gameplayContainer = null;
+            TestDrawableStoryboardSample sample = null;
+
+            OsuModDoubleTime doubleTimeMod = null;
+
+            AddStep("create container", () =>
+            {
+                var beatmap = Beatmap.Value = CreateWorkingBeatmap(new OsuRuleset().RulesetInfo);
+
+                Add(gameplayContainer = new GameplayClockContainer(beatmap, new[] { doubleTimeMod = new OsuModDoubleTime() }, 0));
+
+                SelectedMods.Value = new[] { doubleTimeMod };
+                Beatmap.Value = new TestCustomSkinWorkingBeatmap(beatmap.Beatmap, gameplayContainer.GameplayClock, Audio);
+            });
+
+            AddStep("create storyboard sample", () =>
+            {
+                gameplayContainer.Add(sample = new TestDrawableStoryboardSample(new StoryboardSampleInfo("test-sample", 1, 1))
+                {
+                    Clock = gameplayContainer.GameplayClock
+                });
```

Comment:
Why not merge with above and call that step `setup storyboard sample`, makes sense that way IMO (since we're not testing something with the clock, we just want to setup a working sample).